lib/workers/repository/dependency-dashboard.ts

```diff
@@ -312,7 +314,8 @@ export async function ensureDependencyDashboard(
   }
 
   if (config.dependencyDashboardFooter?.length) {
-    issueBody += `---\n${config.dependencyDashboardFooter}\n`;
+    issueBody +=
+      '---\n' + template.compile(config.dependencyDashboardFooter, config);
```

Comment:
```suggestion
      '---\n\n' + template.compile(config.dependencyDashboardFooter, config);
```

@rarkins I think we should have an additional newline here for better markdown. We had issues in docs without that and it shouldn't harm.